src/sdk/pynni/nni/nas/pytorch/utils.py

```diff
@@ -17,6 +17,14 @@ def global_mutable_counting():
     return _counter
 
 
+def reset_global_mutable_counting():
```

Comment:
global counting is still here?

Reply:
Changed to private method for future refactor (remove global counting entirely)

Reply:
ok